filebeat/input/kafka/kafka_integration_test.go

```diff
@@ -420,6 +429,26 @@ func getTestKafkaHost() string {
 	)
 }
 
+func assertEmpty(t *testing.T, config *common.Config, timeout time.Duration) {
+	client := beattest.NewChanClient(1)
+	defer client.Close()
+	events := client.Channel
+	_, cancel := run(t, config, client)
+	defer cancel()
+
+	timeoutCh := time.After(timeout)
+	select {
+	case event := <-events:
+		if event.Fields == nil && event.Meta == nil && event.Private == nil {
+			// empty event on a close channel
```

Comment:
Could this if block be simplified with the `case event, closed := <-events:` form of a channel received to get the boolean `closed` indicator?

Reply:
This code is not needed anymore, I found out how to do it without reading the events.